relay-common/src/metrics.rs

```diff
@@ -407,6 +404,8 @@ pub trait GaugeMetric {
 }
 
 /// Emits a metric.
+///
+/// See [module-level documentation](metrics/index.html) for examples.
```

Comment:
you can also refer to modules by intra-doc syntax

Reply:
Strange, I tried this in the top-level Relay binary, but it didn't work there. Let me try again.

Reply:
So, I got this to work in cases where the module is imported, but not for extern crates, for instance. Any advice on how to import that?